maistra/bazelrc

```diff
@@ -7,5 +7,5 @@ build --verbose_failures
 # Arch-specific build flags, triggered with --config=$ARCH in bazel build command
 build:x86_64 --linkopt=-fuse-ld=lld
 build:s390x --//source/extensions/filters/common/lua:luajit2=1 --copt="-Wno-deprecated-declarations" --linkopt=-fuse-ld=gold
-build:ppc --//source/extensions/filters/common/lua:luajit2=1 --linkopt=-fuse-ld=lld
+build:ppc64le --//source/extensions/filters/common/lua:luajit2=1 --linkopt=-fuse-ld=lld
```

Comment:
in our build script (maistra/run-ci.sh) we replace `ppc64le` with `ppc`. If we decide not to do this substitution, then we need to remove it from there.

Reply:
No need,  this should be build:ppc

Reply:
@jwendell  changes done in latest commit , can you please review